app/services/participants/ecf.rb

```diff
@@ -14,6 +14,10 @@ def matches_lead_provider?
       relevant_induction_record.present?
     end
 
+    def transferred_induction_record?
+      relevant_induction_record.any?(&:transferred?)
```

Comment:
`relevant_induction_record` returns an AR object or `nil`. so calling `any?` doesn't make sense?

Reply:
No that was left in by accident, will remove